src/Piling/PilingCoordinator.cs

```diff
@@ -1,32 +1,36 @@
 ﻿using Autodesk.Revit.DB;
 using Jpp.Cedar.Core;
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Jpp.Cedar.Piling
 {
     public class PilingCoordinator
     {
-        Definition _eastingDefinition, _northingDefinition, _cutOffDefinition;
+        Definition _eastingDefinition, _northingDefinition, _cutOffDefinition, _permanentLoad, _variableLoad, _windVertical, _windHorizontal;
```

Comment:
Consider explicit access modifier. 

General as a rule I consider implicit declarations are there for the compiler, not for the developer. Failing to explicitly declare the visibility and scope of variables leaves intent ambiguous.